### PR TITLE
May 1 shower

### PR DESCRIPTION
**Pull request by brominger**
I implemented the ScheduleShower

### CODE REVIEW

File: src/main/java/auto_scheduler/StudentScheduleRepositoryImpl.java [hunk 1]
```diff
@@ -18,28 +21,58 @@ public void delete(String studentId) throws FileNotFoundException,  IOException
...
+        if(file.delete()){
+            System.out.println(resourcePathProvider.getClassSchedule() + "/" + studentId + "_schedule.csv deleted");
+        }else System.out.println(resourcePathProvider.getClassSchedule() + "/" + studentId + "_schedule.csv deleted");
     }
 
-    public Collection<String> get(String studentId) throws FileNotFoundException,  IOException {
+    public Collection<Integer> get(String studentId) throws FileNotFoundException,  IOException {
         if(studentId == null || studentId.isEmpty()) 
             throw new IllegalArgumentException("studentid is required!");
+        ArrayList<Integer> sectionNumbers = new ArrayList<>();
```
Comment (samaniehsan) [inline src/main/java/auto_scheduler/StudentScheduleRepositoryImpl.java]: not sure why section nubmbers are integers.

File: src/main/java/auto_scheduler/ScheduleShower.java [hunk 2]
```diff
@@ -1,13 +1,46 @@
...
+            for (CourseInfo course: courses)
+                if(course.getSectionNumber() == sn)
+                    studentSchedule.add(course);
+        }
+        System.out.println(this);
+        return null;
+    }
+
+    @Override
+    public String toString() {
```
Comment (samaniehsan) [inline src/main/java/auto_scheduler/ScheduleShower.java]: @brominger we should implement something like

```java
public class ScheduleSetShow  extends ScheduleSet  {
    @Override
    public String toString() {
	

Attach files by dragging & dropping, selecting or pasting them.

        String classes = "CourseNumber\\CourseName\\SectionNumber\\TimeDay\n";
        for (CourseInfo course: studentSchedule) {
            classes += course.getCourseNumber() + "\\" +
                    course.getCourseName() + "\\" +
                    course.getSectionNumber() + "\\" +
                    course.getDayTime() + "\n";

        }
}
```

File: src/main/java/auto_scheduler/ScheduleShower.java [hunk 3]
```diff
@@ -1,13 +1,46 @@
...
+        CourseScheduleRepositoryImpl classes = new CourseScheduleRepositoryImpl(path);
+        Collection<CourseInfo> c = classes.getClasses();
+        ArrayList<CourseInfo> courses = new ArrayList<>(c);
+        for (Integer sn: sections){
+            for (CourseInfo course: courses)
+                if(course.getSectionNumber() == sn)
+                    studentSchedule.add(course);
+        }
+        System.out.println(this);
+        return null;
```
Comment (samaniehsan) [inline src/main/java/auto_scheduler/ScheduleShower.java]: we should return the results in a class extending `ScheduleSet.`

**Comment by brominger** [comment]
I think I finally get what you want me to do.

**Pull request closed without merging**